odooconnector_base/models/product_supplierinfo.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
actually |product.supplierinfo is defined in product.product so it makes 
sense.
I would segregate though for sales and purchases|